kura/examples/org.eclipse.kura.example.container.signature.validation/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.eclipse.kura.example.container.signature.validation
+Bundle-SymbolicName: org.eclipse.kura.example.container.signature.validation;singletone:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+Service-Component: OSGI-INF/*.xml,
+ OSGI-INF/dummy_signature_validation.xml
+Bundle-ActivationPolicy: lazy
+Import-Package: org.eclipse.kura.container.orchestration;version="1.3.0",
```

Comment:
I think that this should be a range.

Reply:
Addressed in e3afccd9d245416e298fc37d91f4ec40f2eaee41